README.md

```diff
@@ -1505,6 +1505,25 @@ A list of tool extensions for .NET Core Command Line (dotnet CLI), aka '.NET Cor
         <code>dotnet tool install -g <a href="https://www.nuget.org/packages/trx2junit/">trx2junit</a></code>
       </td>
     </tr>
+    <tr>
+      <td><code>try</code></td>
+      <td>
+        <p>
+          Try.NET Global Tool: interactive in-browser documentation and workshop creator
+        </p>
+        <p>
+          <strong>Project site:</strong> <a href="https://dotnet.microsoft.com/platform/try-dotnet">Try .NET</a> <a href="https://github.com/dotnet/try">GitHub</a>
```

Comment:
Can you add a separator between these links, like a comma or something? Visually, these two links run together.